### ISSUE TITLE
[Feature Request] Make luminators reusable

### ISSUE THREAD

**Issue by adudney** [TinyModularThings/IC2Classic#103]
Have them drop either when wrenched or broken with a pick, and have them not steal energy from the electric wrench, perhaps unless shift-right clicking.

**Comment by Speiger** [comment]
O.O luminators are reusable when you wrench it... I mean glass breaks also when using a pick...

**Comment by Carbon-Ghost** [comment]
Wait, weren't they already hand / whatever breakable before ?.... I remember breaking them by hand :P

**Comment by adudney** [comment]
I just tested with IC2 Classic version 1.1.0.19 in survival single player.

I placed a luminator on a tin cable, then tried with three different wrenches.

Regular bronze wrench: Right click and shift right click don't affect it, it merely stays where it was.

Electric wrench in lossless mode: Right click and it flickers on for around 1 tick, then goes back off. Shift right click and nothing happens. No energy is drained from the wrench.

Electric wrench in lossy mode: Right click and it flickers on for around 1 tick, then goes back off. Shift right click and nothing happens. No energy is drained from the wrench.

As a random aside, when a cable is blown up it still visually connects to those around it, even if the cables it's connecting to are placed after it has blown up.

![2015-08-15_17 11 22](https://cloud.githubusercontent.com/assets/246232/9291379/bb4bb2c4-4370-11e5-8c86-fc7b4daf34ae.png)

I placed a luminator on the line, which drew power, above the specified 5 EU/p the cable is rated for, so it exploded.

![2015-08-15_17 11 35](https://cloud.githubusercontent.com/assets/246232/9291380/bb670efc-4370-11e5-97ca-dff91853a736.png)
